decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -159,6 +159,11 @@ def official?
         authors.first.is_a?(Decidim::Organization)
       end
 
+      # Public: Whether the proposal is created in a meeting or not.
+      def official_meeting?
+        authors.first.is_a?(Decidim::Meetings::Meeting)
```

Comment:
```suggestion
        authors.first.class == "Decidim::Meetings::Meeting"
```

This way the proposals component doesn't depend on meetings.

Reply:
:confused: